src/python/grpcio/grpc/_interceptor.py

```diff
@@ -684,57 +684,73 @@ def subscribe(
     def unsubscribe(self, callback: Callable):
         self._channel.unsubscribe(callback)
 
+    # pylint: disable=arguments-differ
     def unary_unary(
         self,
         method: str,
         request_serializer: Optional[SerializingFunction] = None,
         response_deserializer: Optional[DeserializingFunction] = None,
+        registered_method: Optional[bool] = False,
```

Comment:
Please add a test to ensure that `registered_method` gets plumbed through interceptors properly.